migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala

```diff
@@ -139,7 +139,14 @@ final case class JournalMigrator(profile: JdbcProfile)(implicit system: ActorSys
       .forceInsert(journalSerializedRow)
 
     val tagInserts =
-      newJournalQueries.TagTable ++= tags.map(tag => TagRow(journalSerializedRow.ordering, tag)).toSeq
+      newJournalQueries.TagTable ++= tags
+        .map(tag =>
+          TagRow(
+            Some(journalSerializedRow.ordering), // legacy tag key enabled by default.
```

Comment:
This is fine, but we could also read the config key and then decide if we want to fill this column or not.

Ultimately, it will be nicer if users can even not have this column in their table.